packages/editor/src/extensions/table/component.tsx

```diff
@@ -37,6 +37,7 @@ import { getToolDefinition } from "../../toolbar/tool-definitions";
 import { getPosition } from "../../utils/position";
 import { findSelectedDOMNode } from "../../toolbar/utils/prosemirror";
 import { DesktopOnly } from "../../components/responsive";
+import { logger } from "nx/src/utils/logger";
```

Comment:
You can just use `console.log` instead of importing it out of `nx` if required.

Reply:
resolved by 3d59b6e8de9ff09374f26fd5f8cadce99eb50ca5